src/main/java/io/bloco/faker/components/Time.java

```diff
@@ -0,0 +1,71 @@
+package io.bloco.faker.components;
+
+import io.bloco.faker.helpers.Period;
+import java.util.Date;
+import java.util.Random;
+import org.joda.time.DateTime;
+
+public class Time {
+
+  private final Random random;
+
+  public Time() {
+    this.random = new Random();
```

Comment:
I'm using the RandomHelper in FakerComponent for random stuff. You can make it protected and use it here.